docs/adr/0033-content-server-upgrade-to-react.md

```diff
@@ -0,0 +1,140 @@
+
+#### Other Implications
+
+If we want to use GQL everywhere we can instead of hitting auth-server directly with REST API calls, and likely even if we're more selective, we have to ensure that the `graphql-api` can handle:
+
+1. Emitting metrics
+2. Sending emails
+3. Sending push notifications (SQS for event-broker)
+
+- **Option I: Create a new API / set of endpoints in `fxa-auth-server` for `graphql-api`**
+  - Pros: faster to implement
+  - Cons: because it’s more ideal to move functionality into a shared space instead, we’re kicking the can down the road and will eventually need to remove the newly created API in favor of using the shared services
+- **Option II: Move these pieces of functionality into stand-alone library-type services in `fxa-shared` with Typescript**: We can use something like [Bull](https://optimalbits.github.io/bull/) or a job/worker scheme to decouple the service that needs to send an email from the actual email sending service
+  - Pros: this gets us closer to our preferred architecture in the long-term and mitigates more refactoring work later
+  - Cons: Large scope increase for this project
+- **Option III: Combination of I + II**: Depending on level of effort and scope increase, only port over 1-2 of these features into `fxa-shared`. Create the new temporary API for other features to be refactored later.
+  - Pros: this could be a happy medium that allows us to continue moving towards our preferred architecture without blocking work
+  - Cons: Scope increase (less than option II), and still needing refactoring work at a later time (less than option I). May requires us to coordinate plans across teams.
+
+Since Subscription Platform is looking to use GraphQL and move functionality into `graphql-api`, this is also potentially shared work with them.
+
+## Decision Outcomes
+
+### Location of Converted Views
+
+Chosen option: **Option B: Shared React App**, because:
+
+- This is the conventional React approach. A React app can cleanly handle internal routing, and locating all of the views within a single React app is the least confusing pattern for an engineer with work with.
+- Shared state (for flow parameters for example) is much easier to handle if we aren't navigating between different apps
+- We can more easily shared components between pages
+- Navigation between pages will be more efficient
+
+`Positive consequences:` Navigation is faster, the mental model for engineers is simpler (bringing down maintenance complexity), and we don't increase the amount of work to share state across apps.
+`Negative consequences:` We do have to put in the work to update (and safely rollout changes to) the Settings app.
+
+### Incremental Rollout Approach
+
+Chosen option: **Option D: A single experiment with config settings, and no `/beta` route**, because:
```

Comment:
There is a [WIP branch here](https://github.com/mozilla/fxa/compare/main...react-non-beta) that needs some fixing, clean up, and test fixes. If you set your `local.json` config to `showReactApp: { simpleRoutes: true ...}` then `/cannot_create_account?showReactApp=true` shows the React version of the page and content-server `505`s with all that `routeHandlers` stuff that needs edits, if you remove it then content-server `404`s since it doesn't recognize the route. I should be finishing it next sprint, we just wanted to make sure it was plausible.